cpp/src/arrow/compute/api_scalar.h

```diff
@@ -1332,6 +1332,40 @@ Result<Datum> ISOYear(const Datum& values, ExecContext* ctx = NULLPTR);
 ARROW_EXPORT
 Result<Datum> USYear(const Datum& values, ExecContext* ctx = NULLPTR);
 
+/// \brief Year returns PostgreSQL year for each element of `values`
+///
+/// \param[in] values input to extract year from
+/// \param[in] ctx the function execution context, optional
+/// \return the resulting datum
+///
+/// \since 5.0.0
```

Comment:
Why since 5.0.0?

Reply:
I didn't notice this version problem. Which version should it be?

Reply:
https://lists.apache.org/thread/lysnv7bklr56g4jroq2cdr8l2308lfoh

Before freeze it should be mark as 18.0

Reply:
OK, done.